### DISCUSSION TITLE
Farming without wallet

### DISCUSSION THREAD

**Discussion by JackSabbath** [Chia-Network/chia-blockchain#7151] [Q&A] [answered]
I'm farming Chia using my Raspberry Pi. That means the nodes also run on the Raspberry, no it's not just a harvester. I noticed, that the wallet also takes quite a lot of RAM, so I was wondering:

Does farming also work if I run start the system with `chia start farmer-no-wallet`? Or is the wallet required for farming?

edit: Is the Full-Node actually required?

**Comment by loppefaaret** [comment, marked as answer]
to successfully farm, you _need_ the node, the farmer and the harvester - with the new plot NFT, if you use these, your wallet also needs to be synced passed the last time you edited it (switch pool / self pooling)

**Reply by JackSabbath** [reply]
Thank you for the answer.

"synced passed the last time you edited it" -> does that mean that I can turn off the wallet service after it is synced beyond that point (since it would still be synced to that point.. even if it is not up to date)?

**Reply by loppefaaret** [reply]
i make no promises here - it is also new to me, but as far as im aware - the wallet _has_ to be synced passed the last change to the plot NFT - but once it is that, it is no longer needed to be running... please make sure that everything is working if you try it
